docs/DeveloperGuide.md

```diff
@@ -1238,6 +1238,63 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case ends.
 
+**Use case: UC07 - Finding a student**
+
+**MSS**
+
+1. User requests to list students.
+2. TAPA shows a list of students.
+3. User requests to find students from the list.
+4. TAPA displays the students who fit the request.
+
+   Use case ends.
+
+**Extensions**
+
+* 2a. The list is empty.
+
+    Use case ends.
+
+* 3a. The given Student ID is invalid.
+
+    * 3a1. TAPA shows an error message.
+
+      Use case resumes from step 2.
+
+* 3b. The given module code is invalid.
+
+    * 3b1. TAPA shows an error message.
+
+      Use case resumes from step 2.
+
+**Use case: UC08 - Delete all students from a module**
+
+**MSS**
+
+1. User requests to list students.
+2. TAPA shows a list of students.
```

Comment:
If I'm not wrong, this is not required for deleteModule I think? (like I tried find n/x => empty list => deleteModule => still works)